addons/website_event_track/views/event_track_stage_views.xml

```diff
@@ -22,13 +22,13 @@
                         <group>
                             <field name="name"/>
                             <field name="mail_template_id"/>
-                            <field name="sequence" groups="base.group_no_one"/>
+                            <field name="is_visible_in_agenda" attrs="{'readonly': [('is_cancel', '=', True)]}"/>
+                            <field name="is_fully_accessible" attrs="{'readonly': ['|', ('is_cancel', '=', True), ('is_visible_in_agenda', '=', False)]}"/>
+                            <field name="is_cancel"/>
                         </group>
                         <group>
+                            <field name="sequence" groups="base.group_no_one"/>
```

Comment:
If it is in list view we could probably remove it completely, what do you think ?